lib/facter/vas.rb

```diff
@@ -0,0 +1,81 @@
+#vas_usersallow
+Facter.add('vas_usersallow') do
+  setcode do
+    if File.exists?('/etc/opt/quest/vas/users.allow')
+      cmd = 'cat /etc/opt/quest/vas/users.allow 2>/dev/null | tr -d "\r" | sed '/^$/d' | egrep -v "^#" | sort | tr -s "\n" " "'
```

Comment:
This fact now fails silently. I guess it is connected to the use of single ticks in the command. You shouldn't use the same ticks in the command that you have used to specify the command.